src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.cs

```diff
@@ -1385,50 +1417,69 @@ public SolutionState WithDocumentSyntaxRoot(DocumentId documentId, SyntaxNode ro
         /// Creates a new solution instance with the document specified updated to have the source
         /// code kind specified.
         /// </summary>
-        public SolutionState WithDocumentSourceCodeKind(DocumentId documentId, SourceCodeKind sourceCodeKind)
+        public SolutionState WithDocumentSourceCodeKind(
+            ImmutableArray<DocumentId> relatedDocuments,
+            SourceCodeKind sourceCodeKind)
```

Comment:
Why indent args when we have just two or three? Just makes the code consume vertical space unnecessarily.

Reply:
my style is that once i wrap, i wrap them all.